tests/__sf_section_golang.test.fish

```diff
@@ -0,0 +1,100 @@
+source $DIRNAME/spacefish_test_setup.fish
```

Comment:
The tests look great! 🙌 
Thank you for putting in the time to write them!

Reply:
Thank you! Anything I can do to help this awesome project 😊